clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -1,6 +1,41 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.16.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.15.0...0.16.0) (2020-11-26)
```

Comment:
This one should be a bit more compact and doesn't need to include every fix individually. Can some of the less important ones be removed/combined?

Reply:
Ok, makes sense